Sources/Identity.swift

```diff
@@ -48,9 +49,43 @@ import Foundation
 
     // MARK: Event Listeners
 
-    /// Handles events requesting for identifiers. Dispatches response event containing the identifiers. Called by listener registered with event hub.
+    /// Handles events requesting for identifiers. Called by listener registered with event hub.
     /// - Parameter event: the identity request event
     private func handleIdentityRequest(event: Event) {
+        if event.urlVariables {
+            processGetUrlVariablesRequest(event: event)
+        } else {
+            processGetIdentifiersRequest(event: event)
+        }
+    }
+
+    /// Handles events requesting for url variables. Dispatches response event containing the url variables string.
+    /// - Parameter event: the identity request event
+    func processGetUrlVariablesRequest(event: Event) {
+        guard let configurationSharedState = getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.value else { return }
+
+        // if config doesn't have org id we cannot proceed.
+        guard let orgId = configurationSharedState[IdentityConstants.ConfigurationKeys.EXPERIENCE_CLOUD_ORGID] as? String, !orgId.isEmpty else {
+            Log.trace(label: friendlyName, "\(#function) - Cannot process getUrlVariables request Identity event, experienceCloud.org is invalid or missing in configuration ")
```

Comment:
I don't see that we have this check in other platforms. I think we should keep the behavior similar to Identity direct and return the url variable without an org id if one isn't set.

Reply:
From the discussion we would like to keep this check and have orgID as required param.